test/e2e/operator_test.go

```diff
@@ -768,6 +774,7 @@ func TestHostNetworkEndpointPublishingStrategy(t *testing.T) {
 // TestHostNetworkPortBinding creates two ingresscontrollers on the same node
 // with different port bindings and verifies that both routers are available.
 func TestHostNetworkPortBinding(t *testing.T) {
+	t.Parallel()
 	// deploy first ingresscontroller with the default port bindings
 	name1 := types.NamespacedName{Namespace: operatorNamespace, Name: "host"}
```

Comment:
Two problems:
* `TestHostNetworkEndpointPublishingStrategy` and `TestHostNetworkPortBinding` both use the name "host" for the ingresscontrollers that they create.
* If the tests ran in parallel on an on-premise cluster, then they could run out of nodes with available host ports.  A test cluster typically has 3 nodes, the default ingresscontroller with `spec.replicas: 2` would use host ports 80, 443, and 1936 on 2 of the 3 nodes, and that would leave only 1 node with host ports available for tests.

Reply:
Shall we move this to the serial set for now?

Reply:
Yeah, moving it to serial is the safest and simplest thing to do.

Reply:
Moved to serial set: 18f7ed683c5af8c46e679a8d709ec47f1aa54ca4

Reply:
Looks like https://github.com/openshift/cluster-ingress-operator/commit/18f7ed683c5af8c46e679a8d709ec47f1aa54ca4 got inadvertently dropped, resulting in [OCPBUGS-30091](https://issues.redhat.com/browse/OCPBUGS-30091).

Reply:
I posted #1032 to drop `t.Parallel()`.